.github/pull_request_template.md

```diff
@@ -2,7 +2,7 @@ Thanks for submitting a pull request! Please ensure that you have based your cod
 
 - The current stable release branch for bugfixes of limited scope
 - The upcoming [minor](https://semver.org) release branch for backwards compatible changes to existing functionality or addition of new functionality
-- `master` for significant changes to existing behavior
+- `3.x` for significant changes to existing behavior
```

Comment:
This is better, but may make sense to explain diff b/w `2.x` and `3.x` -- 2.x being the default (main) branch.
And 2.x being for developing next 2.* minor version, 3.x for next 3.* minor version.

(esp. for README below)

Reply:
... although this being in 3.x branch itself, maybe it's less confusing to just explain current branch. Your call. :)

Reply:
Very sorry, but I have been extremely busy lately and it has been difficult to check and make corrections (it is Sunday but I am about to go to work).
I think this was the template introduced from `databind`, so I feel that it could be replaced as is.